.github/actions/next-integration-stat/src/index.ts

```diff
@@ -198,23 +128,21 @@ async function getInputs(): Promise<{
     info('No PR number found in context, will not try to post comment.')
   }
 
-  console.log('getInputs: these inputs will be used to collect test results', {
-    token: !!token,
-    shouldDiffWithMain,
-    prNumber,
-    sha,
-    diff_base: getInput('diff_base'),
-  })
-
-  return {
+  const inputs = {
     token,
-    shouldDiffWithMain,
     octokit,
     prNumber,
     sha,
     noBaseComparison,
     shouldExpandResultMessages,
   }
+
+  console.log('getInputs: these inputs will be used to collect test results', {
```

Comment:
Debugging log?

Reply:
This was previously in the `next-integration-stat` command (I just moved the line of code), and IMO it is pretty useful for debugging, so I'd like to leave it here. These logs get collapsed by default in the github actions UI, so it doesn't add much noise if you don't need it.

Reply:
Gotcha!